go.mod

```diff
@@ -6,6 +6,8 @@ replace cloud.google.com/go => cloud.google.com/go v0.102.1
 
 replace github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
 
+replace github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/anujc25/tanzu-plugin-runtime v0.0.0-20231215153217-450decceb82b
```

Comment:
Need to be updated once https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/142 is merged.